src/pydata_sphinx_theme/assets/styles/components/_icon-links.scss

```diff
@@ -2,7 +2,30 @@
  * Icon links in the navbar
  */
 
-.navbar-icon-links {
+.pst-navbar-icon {
```

Comment:
I realized that the `navbar-btn` class isn't defined by Bootstrap but we weren't using it, so I decided to delete navbar-btn everwhere and instead create a new class, `pst-navbar-icon` which I use to apply consistent styling to all the of the icon links and buttons.